db/migrate/20220426235329_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Kindly note that the `Post` table according to the ERD diagram has the `comments_counter` and the `likes_counter`. Kindly implement this. 
- [ ] [REQUIRED] Kindly add the necessary foreign key which will create `author_id`, and a corresponding index. The foreign key is on the ERD diagram.